utopian-io/v2.utopian.io
# Feature/comments

### Verify that
Git checklist

Front end checklist

API checklist

Tests and comments checklist

### **Test plan**
* Go to an article that has comments (http://localhost:8080/en/articles/adriel/another-test-1547586974631)
* Click on the actions button on a comment created by someone else
* **No action should display**
* Click on the actions of a comment that you created. Try to edit and remove
* Reload the page
